cmd/rad/cmd/bicepDownload.go

```diff
@@ -30,7 +28,7 @@ var bicepDownloadCmd = &cobra.Command{
 	Short: "Download the bicep compiler",
 	Long:  `Downloads the bicep compiler locally`,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		output.LogInfo(fmt.Sprintf("Downloading Bicep for channel %s...", version.Channel()))
+		output.LogInfo("Downloading Bicep for channel %s...", version.Channel())
```

Comment:
New version of `golangci-lint` was complaining about these: https://golangci-lint.run/welcome/install/#other-ci.

Reply:
@ytimocin can you separate this change from the typespec update?

Reply:
Just did. I will rebase this once the other PR is in.